### ISSUE TITLE
Implement congestion control

### ISSUE THREAD

**Issue by marcusspangenberg** [Eyevinn/whip-mpegts#1]
From what I can tell, support for the transport-wide-cc header extension (https://datatracker.ietf.org/doc/html/draft-holmer-rmcat-transport-wide-cc-extensions-01) is available in the RTP stats from the dependencies of webrtcbin. 

Try to do something similar as this poc: https://gitlab.freedesktop.org/ystreet/gst-examples/-/commits/bw-management to implement basic congestion control for the sender side.

**Comment by birme** [comment]
Triaged via the daily-backlog-pr workflow. This was an exploratory POC-style request ("try to do something similar") with no acceptance criteria and no dynamic bitrate loop existing today (encoder bitrate is static from config). Broken down into scoped, verifiable pieces:

- #43 — chore: spike transport-wide-cc availability and bandwidth-estimation API in webrtcbin
- #44 — feat: wire GCC bandwidth estimator into the sender pipeline via webrtcbin
- #45 — feat: apply estimated bitrate to the video encoder for sender-side congestion control
- #46 — feat: add opt-in flag and config to enable sender congestion control

Closing in favor of the above; start with the spike (#43) which unblocks the rest.
